src/pybind/mgr/dashboard/frontend/src/app/ceph/dashboard/mgr-summary.pipe.ts

```diff
@@ -15,18 +15,31 @@ export class MgrSummaryPipe implements PipeTransform {
     }
 
     let activeCount = this.i18n('n/a');
-    const titleText = _.isUndefined(value.active_name)
+    const activeTitleText = _.isUndefined(value.active_name)
       ? ''
       : `${this.i18n('active daemon')}: ${value.active_name}`;
-    if (titleText.length > 0) {
+    if (activeTitleText.length > 0) {
       activeCount = '1';
     }
+    let str = '';
+    let flg = false;
+    for (const i in value.standbys) {
+      if (flg) {
+        str = str + ', ' +  value.standbys[i].name;
+      } else {
+        str = str + value.standbys[i].name;
+        flg = true;
+      }
+    }
```

Comment:
Same like in `mds-summary`
```suggestion
    const str = value.standbys.map((s: any): string => s.name).join(', ');
```

Reply:
standbys doesn't have 'map' property, will show undefined

Reply:
See my other comment